llvm-external-projects/iree-dialects/lib/Dialect/LinalgExt/Passes/TileAndDecomposeAttention.cpp

```diff
@@ -427,6 +440,9 @@ IREE::LinalgExt::AttentionOp tileAttention(IREE::LinalgExt::AttentionOp attnOp,
       SmallVector<Value>{querySlice, keySlice, valueSlice},
       SmallVector<Value>{iterArgResult, iterArgMax, iterArgSum});
 
+  if (attnOp.getTransposeV())
+    tiledAttentionOp->setAttr("transpose_v", rewriter.getBoolAttr(true));
```

Comment:
nit: you should be able to use `attnOp.getTransposeVAttrName` instead of the string literal, or just `attnOp.setTransposeVAttr`.

Reply:
thanks! that's much better.